src/lib/bootloader/device_map.rb

```diff
@@ -57,11 +57,7 @@ def contain_disk?(disk)
     end
 
     def disks_order
-      grub_devices = @model.grub_devices
-      grub_devices.select! { |d| d.start_with?("hd") }
-      grub_devices.sort_by! { |dev| dev[2..-1].to_i }
-
-      grub_devices.map { |d| @model.system_device_for(d) }
+      sorted_disks.map { |d| @model.system_device_for(d) }
     end
```

Comment:
it is even more tricky, as it is sorted grub devices, but then it is too long method for me...and it is not just grub devices, but grub devices representing disks ( we filter out all non hd devices ). So I expect that good name is very hard to find.